wiki/markdown/compatibility.md

```diff
@@ -1,38 +1,53 @@
 # Compatibility
 
-We recommend using the latest version of back-end and front-end whenever possible. However, this page contains a compatibility matrix and notes when pairing different versions.
-
-|                      |Front-end ➡           |                      |                      |                      |
-|---------------------:|:--------------------:|:--------------------:|:--------------------:|:--------------------:|
-|**Back-end ⬇**        |                 1.0.0|                 1.0.1|                 1.0.2|                 1.0.3|
-|[1.0.0](#back-end-100)|[✔️️](#front-end-100)   |[✔️️](#front-end-101)   |[✅](#front-end-102)  |[✅](#front-end-103)  |
-|[1.0.1](#back-end-101)|[✔️️](#front-end-100-1) |[✔️️](#front-end-101-1) |[✅](#front-end-102-1)|[✅](#front-end-103-1)|
-|[1.0.2](#back-end-102)|[❕](#front-end-100-2)|[✅](#front-end-101-2)|[✔️️](#front-end-102-2) |[✔️️](#front-end-103-2) |
+We recommend using the latest version of back-end and front-end whenever 
+possible. However, this page contains a compatibility matrix and notes 
+when pairing different versions.
+
+|                      |Recommended front-end versions  |
+|---------------------:|:------------------------------:|
+|**Back-end ⬇**        |                                |
+|[2.0.0](#back-end-200)|Front-end 2.0.0 version         |
+|[1.0.2](#back-end-102)|Front-end 1.x versions          |
+|[1.0.1](#back-end-101)|Front-end 1.x versions          |
+|[1.0.0](#back-end-100)|Front-end 1.x versions          |
 
 ✔️️ Yes | ✅ Yes, minor Issue | ❕ Yes, medium Issue | ❌ No, major Issue
 
-## Back-end: 1.0.0
-#### Front-end: 1.0.0
+## Back-end: 2.0.0
+#### Front-end: 2.0.0
 Compatible: ✔️️ Yes
 
-Notes: -
+#### Front-end: 1.x
+Compatible: ❌  No, major Issue
 
-#### Front-end: 1.0.1
-Compatible: ✔️️ Yes
+Notes: 
+- In Eiffel Intelligence 2.0.0 some new changes were introduced which
+are not backwards compatible. [See release notes](https://github.com/eiffel-community/eiffel-intelligence-frontend/releases/tag/2.0.0).
 
-Notes: -
+## Back-end: 1.0.2
+#### Front-end: 1.0.0
```

Comment:
In Back-end: 2.0.0 section front-ends are listed from newest to oldest but in the older back-end versions they are still from oldest to newest.

Reply:
But maybe this does not matter. I don't know.

Reply:
Changed the order of front-ends